data/sql/updates/pending_db_world/blightguard.sql

```diff
@@ -0,0 +1,45 @@
+-- Delete all Alarmed Blightguard, they should be spawned during quest event
+DELETE FROM `creature` WHERE (`id1` = 28745);
+
+-- Update Blightguard AI
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE (`entry`=28603);
+
+-- Add missing skill to Blightguard
```

Comment:
Can you maybe add another comment with spell id + spell name please

Reply:
Now I'm no longer sure if I need to do this part. The spell is cast via Smart AI. Do we still need to add it here?

Reply:
Well this table holds the spells that the creature has wen beeing mind controlled.
It's not necessary for anything else, so irrelevant for SAI for example.